packages/wallet/backend/src/app.ts

```diff
@@ -185,6 +185,11 @@ export class App {
       isAuth,
       paymentPointerController.registerKey
     )
+    router.patch(
```

Comment:
maybe use delete, and instead of revoke-key -> keys

Reply:
At the moment we only support one key per payment pointer, that's why I went for `PATCH` and `revoke-key`.